RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforDevice.h

```diff
@@ -234,17 +234,30 @@ namespace pixelCPEforDevice {
     if (cp.minCol[ic] == 0 || cp.maxCol[ic] == uint32_t(detParams.nCols - 1))
       cp.ysize[ic] = -cp.ysize[ic];
 
+    /* original code, kept for documentation
+
```

Comment:
then delete it!

Reply:
maybe add "make sure optimization does not produce different code on different architectures "
(or similar)